eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -124,9 +124,7 @@ jobs:
       - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(parameters.jobParameters.helixQueueGroup, 'pr', 'ci', 'libraries')) }}:
         - Windows.10.Amd64.Open
       - ${{ if and(eq(variables['System.TeamProject'], 'public'), notIn(parameters.jobParameters.helixQueueGroup, 'pr', 'ci', 'libraries')) }}:
-        - (Windows.Nano.1809.Amd64.Open)windows.10.amd64.serverrs5.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1809-helix-amd64-08e8e40-20200107182504
```

Comment:
Is Nano Server out of support, or just the underlying RS5 image? Can we work on updating the image to keep running tests on Nano Server in rolling or nightly builds?

Reply:
I ask because when I re-enabled this coverage in my PR, I found a few test failures that we needed to fix since Nano Server doesn’t support some of the features we use in some of our tests (like the STA or Reg Free COM)

Reply:
You are right, Windows Server RS5 (1809) should be removed. I'll revert this deletion